packages/superset-ui-chart-controls/src/sections/advancedAnalytics.tsx

```diff
@@ -126,5 +126,61 @@ export const advancedAnalyticsControls: ControlPanelSectionConfig = {
         },
       },
     ],
+    [<h1 className="section-header">{t('Resample')}</h1>],
+    [
+      {
+        name: 'resample_rule',
+        config: {
+          type: 'SelectControl',
+          freeForm: true,
+          label: t('Rule'),
+          default: null,
+          choices: [
+            ['1T', '1 minutely frequency'],
+            ['1H', '1 hourly frequency'],
+            ['1D', '1 calendar day frequency'],
+            ['7D', '7 calendar day frequency'],
+            ['1MS', '1 month start frequency'],
+            ['1M', '1 month end frequency'],
+            ['1AS', '1 year start frequency'],
+            ['1A', '1 year end frequency'],
+          ],
+          description: t('Pandas resample rule'),
+        },
+      },
+    ],
+    [
+      {
+        name: 'resample_method',
+        config: {
+          type: 'SelectControl',
+          freeForm: true,
+          label: t('Method'),
+          default: null,
+          choices: [
+            ['asfreq', 'Null values fill'],
+            ['ffill', 'Forward values fill'],
+            ['bfill', 'Backward values fill'],
+            ['median', 'Median values fill'],
+            ['mean', 'Mean values fill'],
+            ['sum', 'Sum values fill'],
+          ],
+          description: t('Pandas resample method'),
```

Comment:
Maybe we could call this "Fill method" and then remove the word "fill" from the choices, like "Forward values fill" would become "Forward values". Also, could we replace "Null values fill" and the potentially forthcoming "Zero values fill" with "Null imputation" and "Zero imputation"?

Reply:
Nice suggestion!